tests/mock-e2e/agent/agent_test.go

```diff
@@ -1,4 +1,4 @@
-package agent_test
+package agent
```

Comment:
I don't see this move as productive, this test was created to begin the process of adding more tests. To better understand how the components work together. Moving our main agent test out of agent as the only integration test feels premature.

Reply:
Ok, as per https://github.com/flightctl/flightctl/tree/main/test strategy, the final directory would be /test/integration/agent